tests/mediator.ts

```diff
@@ -13,6 +14,29 @@ import { DidCommMimeType } from '../src/types'
 
 import config from './config'
 
+const logger = testLogger
+
+class HttpTransportSession implements TransportSession {
```

Comment:
Why is `WebSocketTransportSession` part of the framework, but `HttpTransportSession` not?

Reply:
Ah, good point. I thought it's gonna be used inside `WsOutboundTransporter`. I moved `WebSocketTransportSession` to `mediator-ws` and we have it in the same place now. Eventually, it can go inside framework together with those inbound transports. (I can move it there when I'll be renaming transporters to transport)